contents/handbook/growth/sales/who-we-do-business-with.md

```diff
@@ -43,3 +43,15 @@ US sanctions mean that we are not allowed to offer services [at all](https://www
 - North Korea
 - Syria
 
+## Update for June 2024 US sanctions against Russia 
+In June 2024, the US Treasury's Office of Foreign Asset Control issued [updated sanctions against Russia](https://ofac.treasury.gov/sanctions-programs-and-country-information) which prohibit the sale or supply of services to individuals or organizations in Russia. The sanctions take effect on September 10, 2024 and continue indefinitely.
+
+We must comply with these sanctions, so in August 2024 we contacted impacted individuals to let them know we would make the following changes on **September 9th, 2024**:
+
+- We no longer accept any payments from individuals or organizations based in Russia
+- We block access to PostHog for all individuals in Russia, based on their IP
+- We terminated paid accounts with all customers located in Russia
+
+Under the new sanctions organizations based in Russia but controlled by US citizens would be exempt, but individuals based in Russia are subject to sanctions even if working for US-based organization.
```

Comment:
Sorry just catching this, I am not 100% the latter part is true in regards to individuals even if working in Russia for us-based organisations. I think there is an exclusion to the sanctions for this. I would maybe change this line to something like. 

"There are some exemptions to the sanctions including any service to any entity located in the Russian Federation that is owned or controlled, directly or indirectly, by a U.S. person."

Reply:
```suggestion
There are some exemptions to the sanctions including any service to any entity located in the Russian Federation that is owned or controlled, directly or indirectly, by a U.S. person.
```